dcos/dcos-ui
# DCOS-49571: Services Table > Resume Service modal has an incomplete error description

Add a readable error for a negative instance count
in service actions modals.

Closes https://jira.mesosphere.com/browse/DCOS-49571

## Testing
1. Go to services tab.
2. Start a service if there are none.
3. Open the scale modal for that service.
4. Enter a negative number and submit.
5. Verify that the new readable error message is shown.
6. Stop the service.
7. Open the resume modal.
8. Enter a negative number and submit.
9. Verify that the new readable error message is shown.
10. Verify that the added tests make sense.

## Trade-offs
1. I wondered if it's worth it adding an util, but I think we might expand it in the future.
2. I wondered about the function name. Suggestions are appreciated.
3. I tried to add an integration test for this, but cypress refused to type a minus.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-08-15 11-27-10](https://user-images.githubusercontent.com/40791275/63082965-39d41500-bf50-11e9-85ed-1d02d6ebc7fb.png)

### After
![Screenshot from 2019-08-15 11-09-46](https://user-images.githubusercontent.com/40791275/63082967-3ccf0580-bf50-11e9-8e57-568fd04049e3.png)
